test/rules/url.spec.js

```diff
@@ -17,7 +23,7 @@ describe("Test rule: url", () => {
 		expect(check(true)).toEqual([{ type: "string", actual: true, message }]);
 
 		message = "The '' field must be a valid URL.";
-		expect(check("")).toEqual([{ type: "url", actual: "", message }]);
+		expect(check("")).toEqual(true);
```

Comment:
Good job. But my problem is that it means it's a breaking change.

Reply:
@icebob And I think It's not good to allow empty string by default

Reply:
@icebob @erfanium Hmm, I see your point of view. The main reason I've set the `schema.empty` as `true` by default was because that's the case for every rule.

Would you prefer to be `false` by default?